src/redturtle/prenotazioni/utils/migration.py

```diff
@@ -0,0 +1,7 @@
+from zope.globalrequest import getRequest
+from collective.exportimport.interfaces import IMigrationMarker
```

Comment:
metti un try except ImportError qui e togli il requirement da setup.py

Reply:
Done, anche un mock in piu